cfme/tests/control/test_alerts.py

```diff
@@ -286,36 +300,35 @@ def test_alert_timeline_cpu(request, vm_name, provider, ssh, vm_crud):
             "Real Time Performance",
             {
                 "performance_field": "CPU - % Used",
-                "performance_field_operator": ">",
-                "performance_field_value": "20",
+                "performance_field_operator": ">=",
+                "performance_field_value": "10",
                 "performance_trend": "Don't Care",
                 "performance_time_threshold": "2 Minutes",
             }),
-        notification_frequency="5 Minutes",
+        notification_frequency="1 Minute",
         timeline_event=True,
     )
     alert.create()
     request.addfinalizer(alert.delete)
 
-    setup_for_alerts(request, [alert])
-    # Generate a 100% CPU spike for 5 minutes, that should be noticed by CFME.
-    ssh.cpu_spike(seconds=5 * 60, cpus=4)
-
-    def _timeline_event_present():
-        vm_crud.open_timelines()
-        select = pytest.sel.Select("//select[@name='tl_fl_grp2']")  # TODO: Make a timelines module?
-        pytest.sel.select(select, "Alarm/Status change/Errors")
-        for event in jstimelines.events():
-            info = event.block_info()
-            if info.get("Event Type") != "EVMAlertEvent":
-                continue
-            if info.get("Event Source") != "MiqAlert":
-                continue
-            if info["Source VM"] == vm_name:
-                return True
-        return False
-
-    wait_for(_timeline_event_present, num_sec=30 * 60, delay=15, message="timeline event present")
+    setup_for_alerts(request, [alert], vm_name=vm_name)
+    # Generate a 100% CPU spike for 15 minutes, that should be noticed by CFME.
+    ssh.cpu_spike(seconds=60 * 15, cpus=2, ensure_user=True)
+    vm_crud.open_timelines()
```

Comment:
REQUIRED: please use view = navigate_to(vm, 'Timelines') instead of all this staff